### ISSUE TITLE
Mutiple simultaneous crawls causes lock errrors

### ISSUE THREAD

**Issue by mitra42** [internetarchive/dweb-mirror#109]
Test - run two crawls at the same time (different items)
Different errors seen ...

**Comment by mitra42** [comment]
```
  dweb-archivecontroller:Util fetch_json: https://dweb.me/arc/archive.org/metadata/thetaleofpeterra14838gut/14838-h%2Fimages%2Fpeter51.jpg +297ms
(node:13491) UnhandledPromiseRejectionWarning: ReadError: Database is not open
    at maybeError (/Users/mitra/git/dweb-mirror/node_modules/level-packager/node_modules/levelup/lib/levelup.js:293:32)
    at LevelUP.put (/Users/mitra/git/dweb-mirror/node_modules/level-packager/node_modules/levelup/lib/levelup.js:187:7)
    at HashStore.f (/Users/mitra/git/dweb-mirror/HashStore.js:51:33)
    at Promise (/Users/mitra/git/dweb-mirror/HashStore.js:39:124)
    at new Promise (<anonymous>)
    at HashStore.put (/Users/mitra/git/dweb-mirror/HashStore.js:39:81)
    at fs.rename (/Users/mitra/git/dweb-mirror/MirrorFS.js:386:92)
    at FSReqWrap.oncomplete (fs.js:141:20)
(node:13491) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 5)
  dweb-transports:httptools Delaying fetching https://dweb.me:443/contenthash/5dttHsY2AZx5fGN6tZooR8tmvXC36C by 500 ms because request to https://dweb.me:443/con
```

**Comment by mitra42** [comment]
I think this one happens first, and then the error above occurs because we neither loop retrying this, nor detect that it failed.
```
 dweb-transports Opening stream to https://be-api.us.archive.org/mds/v1/get_related/all/thetaleofpeterra14838gut via HTTP +13s
(node:13491) UnhandledPromiseRejectionWarning: OpenError: IO error: lock /Volumes/Transcend/archiveorg/.hashStore.sha1.relfilepath/LOCK: Resource temporarily unavailable
    at /Users/mitra/git/dweb-mirror/node_modules/level-packager/node_modules/levelup/lib/levelup.js:87:23
    at /Users/mitra/git/dweb-mirror/node_modules/abstract-leveldown/abstract-leveldown.js:41:14
    at /Users/mitra/git/dweb-mirror/node_modules/level-packager/node_modules/deferred-leveldown/deferred-leveldown.js:20:21
    at /Users/mitra/git/dweb-mirror/node_modules/abstract-leveldown/abstract-leveldown.js:41:14
    at /Users/mitra/git/dweb-mirror/node_modules/abstract-leveldown/abstract-leveldown.js:41:14
(node:13491) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:13491) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
  dweb-transports Opening stream to https://be-api.us.archive.org/mds/v1/get_related/all/thetaleofpeterra14838gut via HTTP succeeded +1ms
  ```

**Comment by mitra42** [comment]
This may have been fixed by the #102 fix, so closing as haven't seen again
